### PR TITLE
Tagalog

### PR DESCRIPTION
**Pull request by timoballard**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/204)

## What does this change?
Adds Tagalog translations for public-facing site content.

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [x] Check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [x] Re-check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

**Comment by timoballard** [comment]
@Jkrzy @ilodidoj - this is ready for review!

File: crt_portal/cts_forms/locale/tl/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,2734 @@
...
+"          %(word_limit)s word limit reached\n"
+"        "
+msgstr ""
+"\n"
+"          %(word_limit)s naabot na ang hangganan ng salita\n"
+"        "
+
+#: templates/landing.html:29
+msgid "Menu"
+msgstr ""
```
Comment (Jkrzy) [inline crt_portal/cts_forms/locale/tl/LC_MESSAGES/django.po:1723]: Flagging missing translation

File: crt_portal/cts_forms/locale/tl/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,2734 @@
...
+msgid "Budget and performance"
+msgstr "Badyet at pagganap"
+
+#: templates/partials/footer.html:96
+msgid "Legal policies and Disclaimers"
+msgstr "Ligal na mga patakaran at mga Pagtanggi"
+
+#: templates/partials/i8n.html:15
+msgid "Go"
+msgstr ""
```
Comment (Jkrzy) [inline crt_portal/cts_forms/locale/tl/LC_MESSAGES/django.po:2196]: Flagging missing translation

**Review by Jkrzy — CHANGES_REQUESTED** [review]
Looks great @timoballard! Only thing to address are two missing translations.

**Review by Jkrzy — APPROVED** [review]
Thanks @timoballard, we'll need some styling tweaks for the "Go" button in the footer but can tackle them in https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/932

**Pull request merged**